### PR TITLE
Create A/B test for new user dashboard tile size

### PR DESCRIPTION
**Pull request by NoahCarnahan**
FYI review buddy @czue, @orangejenny.

@biyeun this is my first Angular PR, so would love some feedback if I went about this in a totally unconventional way.

### CODE REVIEW

File: corehq/apps/style/static/dashboard/less/dashboard.less [hunk 1]
```diff
@@ -108,4 +108,8 @@
   }
 }
 
+[ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
+  display: none !important;
```
Comment (NoahCarnahan) [inline corehq/apps/style/static/dashboard/less/dashboard.less]: This prevents flicker in the time between the initial page load and the first angular render.
https://docs.angularjs.org/api/ng/directive/ngCloak

Comment (orangejenny) [inline corehq/apps/style/static/dashboard/less/dashboard.less]: could this comment go in the LESS itself? would be useful for future devs.

Comment (NoahCarnahan) [inline corehq/apps/style/static/dashboard/less/dashboard.less]: oh yeah, that makes more sense :)

Comment (NoahCarnahan) [inline corehq/apps/style/static/dashboard/less/dashboard.less]: (updated)

**Comment by czue** [comment]
don't feel qualified to review this, so will defer to @biyeun

File: corehq/apps/dashboard/templates/dashboard/dashboard_new_user.html [hunk 2]
```diff
@@ -21,6 +21,37 @@
 </script>
 {% endblock js-inline %}
 
+{% block uncompressed-js %} {{ block.super }}
+    <script type="text/javascript">
+        (function (angular, undefined) {
+            'use strict';
+            var dashboardApp = angular.module('dashboardApp', []);
```
Comment (biyeun) [inline corehq/apps/dashboard/templates/dashboard/dashboard_new_user.html]: While this is ok to do everything in the page for an a/b test (as the code is temporary) if code grows at all or will have long term support, it's better to add it in an external file. Best practices for doing so are outlined VERY well in this video, and I highly recommend you give it the hour to watch (or speed it up 2x for a half hour): https://www.youtube.com/watch?v=i9MHigUZKEM

For now inline JS is ok (as it's an a/b test). But please put a comment at the top that this style should not be copied in the rest of the code base.

**Comment by biyeun** [comment]
left a comment. will merge after the comment is added to the inline js :)

**Comment by NoahCarnahan** [comment]
Thanks for the feedback. Added a note as recommended.

**Comment by orangejenny** [comment]
hooray!

**Pull request merged**